app/js/controllers/foyer/individuForm.js

```diff
@@ -122,6 +123,19 @@ angular.module('ddsApp').controller('FoyerIndividuFormCtrl', function($scope, in
         }
     };
 
+    $scope.capturePerteAutonomie = false;
+
+    /* We need to delay the apparition of the 'perteAutonomie' question,
```

Comment:
I'm afraid “apparition” is a _faux-ami_, “delay displaying” would probably be more appropriate  ;)